src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -314,12 +311,6 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps, GraphSetting
     ];
 
     const badgeOptions: DisplayOptionType[] = [
-      {
-        id: 'filterCB',
-        labelText: 'Circuit Breakers',
-        isChecked: showCircuitBreakers,
-        onChange: toggleGraphCircuitBreakers
-      },
```

Comment:
This removal is fine, but now that the `Virtual Service` option is more complex/powerful, in that it is used to render not only VS but also the related scenario badges, it deserves an info tooltip.  See above for an example of adding the `tooltip` attribute to the option.